.github/ISSUE_TEMPLATE/default_template.md

```diff
@@ -0,0 +1,24 @@
+<!-- Fill out this issue by completing the items below. Not every item may be relevent to you, so feel free to make changes-->
+
+#### The current behavior or issue
+
+Description:
+
+#### The steps taken to reproduce the behavior or issue, or specify a location where the steps were recorded
+
+To reproduce:
+
+#### Describe the desired behavior or desired outcome
+
+Goal:
+
+#### Add other supporting information that may be useful
+
+Versions, OS information, other:
+
+#### Completion criteria
+
+This is done when:
+- [ ] (replace this text)
+- [ ] documentation updated
```

Comment:
Added tests + docs to completion criteria.

Reply:
Thanks for the change